### DISCUSSION TITLE
ΑΝΤΙΜΕΤΩΠΙΖΩ ΘΕΜΑ ΣΤΟ 7 ΜΕ ΤΟ NETLIFY

### DISCUSSION THREAD

**Discussion by StefanosSfinarolakis** [ContattoContare/site#28] [Q&A] [answered]
Δεν μπορώ να κάνω deploy επειδή μου βγάζει κάτι λάθος στο includes......          *plaese help me*
https://github.com/StefanosSfinarolakis/site/blob/master/_includes/cs-joystick.md

**Comment by NikosTrypakis** [comment, marked as answer]
Παρατήρησα ότι μέσα στο [includes](https://github.com/StefanosSfinarolakis/site/blob/master/_includes/cs-joystick.md) σου, στο Figure 1 υπάρχει ένα παραπάνω εισαγωγικό (") 

`...Figure 1:"Σε αντίθεση...`

Δοκίμασε να το βγάλεις και κάνε ξανά deploy το site σου.

**Reply by StefanosSfinarolakis** [reply]
ty <3

**Comment by p18balt** [comment]
Σου δούλεψε αν όχι δοκίμασε να κάνεις αλλαγές στο _config.yml και να βάλεις το προσωπικό λινκ του νετλιφυ και το προσωπικό σου repo στο site.
